src/Datadog.Trace/DiagnosticListeners/AspNetCoreDiagnosticObserver.cs

```diff
@@ -73,6 +76,13 @@ public AspNetCoreDiagnosticObserver(Tracer tracer, Security security)
             _security = security;
         }
 
+        public AspNetCoreDiagnosticObserver(string httpOperationName, IntegrationInfo integrationInfo)
```

Comment:
Is this overload being called anywhere?

Reply:
Ah yes, had a local change from the reapply over master I hadn't committed, good catch.